app/src/main/java/org/literacyapp/receiver/BootReceiver.java

```diff
@@ -73,11 +73,13 @@ public static void scheduleMergeSimilarStudents(Context context){
         JobInfo.Builder builderMergeSimilarStudents = new JobInfo.Builder(LiteracyApplication.MERGE_SIMILAR_STUDENTS_JOB_ID, componentNameMergeSimilarStudents);
         boolean requiresCharging = true;
         builderMergeSimilarStudents.setRequiresCharging(requiresCharging);
+        boolean requiresDeviceIdle = true;
+        builderMergeSimilarStudents.setRequiresDeviceIdle(requiresDeviceIdle);
```

Comment:
The recurrence is handled by the setPeriodic method. So I think it will be triggered once in Idle mode and then rescheduled 24hrs afterwards (maybe we should set this to 12hrs). Since idle mode is a loose definition I wan't able to test it really until now.

Another option we discussed was that we define a time (say 30 minutes) after the screen has been turned of to call it idle from our side. I don't know what it better.